src/System.IO.Ports/tests/SerialPort/BreakState.cs

```diff
@@ -12,7 +12,7 @@ namespace System.IO.Ports.Tests
     public class BreakState_Property : PortsTest
     {
         //The maximum time we will wait for the pin changed event to get firered for the break state
-        static readonly int MAX_WAIT_FOR_BREAK = 800;
+        private static readonly int s_MAX_WAIT_FOR_BREAK = 800;
```

Comment:
Could be. Maybe go back to before the formatting, change to const, then rerun it? Perhaps it's just resistant to removing s_ but will then not actually add it.